src/createRollupConfig.ts

```diff
@@ -15,6 +16,15 @@ import { babelPluginTsdx } from './babelPluginTsdx';
 import { TsdxOptions } from './types';
 import * as fs from 'fs-extra';
 
+const USE_COMPILER = Boolean(process.env.CLOSURE_COMPILER);
```

Comment:
why not use a CLI flag? microbundle has a similar discussion going in https://github.com/developit/microbundle/pull/569

Reply:
fixed